### ISSUE TITLE
ng update error from rc.1

### ISSUE THREAD

**Issue by kolkov** [angular/angular-cli#16190]
From previous RC.1
![image](https://user-images.githubusercontent.com/3740898/68857656-52410080-06f4-11ea-8557-52674f66b974.png)
![image](https://user-images.githubusercontent.com/3740898/68857839-c7143a80-06f4-11ea-80f7-3a0c14df4836.png)

```
[error] Error: spawnSync npm ENOENT
Package install failed, see above.
    at installPackage (D:\Go\libs\fcui\node_modules\@angular\cli\tasks\install-package.js:39:15)
    at installTempPackage (D:\Go\libs\fcui\node_modules\@angular\cli\tasks\install-package.js:62:5)
    at Object.runTempPackageBin (D:\Go\libs\fcui\node_modules\@angular\cli\tasks\install-package.js:67:33)
    at UpdateCommand.run (D:\Go\libs\fcui\node_modules\@angular\cli\commands\update-impl.js:153:38)
    at async UpdateCommand.validateAndRun (D:\Go\libs\fcui\node_modules\@angular\cli\models\command.js:134:28)
    at async Object.runCommand (D:\Go\libs\fcui\node_modules\@angular\cli\models\command-runner.js:201:24)
    at async default_1 (D:\Go\libs\fcui\node_modules\@angular\cli\lib\cli\index.js:54:31)
```

**Comment by elvisbegovic** [comment]
You have to be on `9.0.0-rc.2` to get that issue fixed.

cc @alan-agius4

**Comment by alan-agius4** [comment]
Yeah, this was fixed in `9.0.0-rc.2`.


Please try
```
npm i @angular/cli@9.0.0-rc.2 --no-save
ng update --next
```

**Comment by kolkov** [comment]
Save this worked!
